src/makielayout/blocks.jl

```diff
@@ -342,8 +342,8 @@ function _block(T::Type{<:Block}, fig_or_scene::Union{Figure, Scene},
     layout_height = Observable{Any}(nothing)
     layout_tellwidth = Observable(true)
     layout_tellheight = Observable(true)
-    layout_halign = Observable{Union{Symbol, Float64}}(:center)
-    layout_valign = Observable{Union{Symbol, Float64}}(:center)
+    layout_halign = Observable{Union{Symbol, Real}}(:center)
```

Comment:
I'd rather improve the conversion instead of making the type more abstract...At least if easily possible ;)

Reply:
I was thinking the same. I think for most of these cases using a Union is not enough as you can't use it for dispatch that well. Probably using small descriptive container types is better so that we can define conversions on those.

Reply:
Just an elaboration. IIUC, then this appears at the root of the Block definition, which lead me to think that this might also potentially fix the problem for other elements too. (premature bug-fix? ;) )

But I do see what you mean.

Just wanted to mention, unless I am wrong, `Union{Symbol, Float64}` is not _more_ abstract than `Union{Symbol, Real}`. But I understand that using `Union{Symbol, Real}` here will introduce the abstract `Real` somewhere later down the pipeline.

I will change it.